callstack/pom.xml

```diff
@@ -30,10 +30,6 @@
     <module>org.eclipse.tracecompass.incubator.analysis.core</module>
     <module>org.eclipse.tracecompass.incubator.analysis.core.tests</module>
     <module>org.eclipse.tracecompass.incubator.callstack</module>
```

Comment:
Is there a specific reason why the incubator callstack feature was not removed ?